IS7024/Pages/HollywoodBowl.cshtml

```diff
@@ -41,29 +45,18 @@
                     var newhour = Convert.ToInt32(mhour);
                     int realhour = newhour - 12;
                     string starttime = realhour + ":" + mmin + " PM";
-
-
-
-                <tr>
+                    <tr>
                         <td>@day/@month/@yearclean</td>
                         <td>@eventss.Performance[0].Artist.DisplayName</td>
                         <td>@eventss.Status</td>
                         <td>@starttime</td>
                         <td>@sun</td>
                         <td><a href="@eventss.Uri" target="_blank">Get Tickets</a></td>
-
-                </tr>
-                    }
+                    </tr>
                 }
-
             }
-       
-
-
-
-
-
+        }
     </table>
-    <img src="https://raw.githubusercontent.com/huberr3/IS7024/master/powered-by-songkick-pink.png" width="171" height="60">
+    <img alt ="Songkick Logo" src="https://raw.githubusercontent.com/huberr3/IS7024/master/powered-by-songkick-pink.png" width="171" height="60">
```

Comment:
Added "alt" to the image so that if the image doesn't load it states what the image is.

Reply:
Definitely a good idea.  This is required for accessibility.